install/kubernetes/cilium/values.yaml

```diff
@@ -712,6 +712,9 @@ ingressController:
   # Incoming traffic to http listener will return 308 http error code with respective location in header.
   enforceHttps: true
 
+  # -- Enable proxy protocol for the listeners.
+  enableProxyProtocol: false
```

Comment:
Please report this change to `install/kubernetes/cilium/values.yaml.tmpl` and then run

```
make -C install/kubernetes cilium/values.yaml && make -C Documentation update-helm-values
```

Reply:
@kaworu done